axie-scholar-utilities/source/axie/payments.py

```diff
@@ -240,8 +239,8 @@ def verify_inputs(self):
             self.type = "new"
         except ValidationError as ex:
             new_msg = ("If you were tyring to pay using the current format:\n"
-                          f"Error given: {ex.message}\n"
-                          f"For attribute in: {list(ex.path)}\n")
+                       "Error given: {ex.message}\n"
```

Comment:
forgot the f', this would format it wrong

Reply:
yes, you are right, fixing now